src/main/java/Group15/Api/Exercise.java

```diff
@@ -8,6 +8,8 @@ public class Exercise {
     public String equipment;
     public String difficulty;
     public String imagePath;
+    public int timePerRep;
+
 
     public Exercise(String title, String description, String type, String bodyPart, String equipment, String difficulty, String imagePath) {
```

Comment:
Either add timePerRep to this constructor or rewrite tests so that they dont use this.
I actually like your way better (creating empty object and setting values line for line)

Reply:
So what do you want me to do?